cmd/kube-proxy/app/server_others_test.go

```diff
@@ -150,7 +167,11 @@ func Test_getProxyMode(t *testing.T) {
 			modules:       c.kmods,
 			kernelVersion: c.kernelVersion,
 		}
-		canUseIPVS, _ := ipvs.CanUseIPVSProxier(khandler, ipsetver)
+		scheduler := cases[i].scheduler
+		if scheduler == "" {
```

Comment:
This feels like a behavior that should be done in `CanUseIPVSProxier` (or somewhere else) instead of in the test

Reply:
It is currently defaulted in [NewProxier](https://github.com/kubernetes/kubernetes/blob/86096addb146c740ace72daf1291f81ef55490f2/pkg/proxy/ipvs/proxier.go#L433-L436). I can put in another one of those checks if needed.

Reply:
Ah I see, seems like passing an empty scheduler is an error case for CanUseIPVSProxier then? Can we update these tests to pass in "" to CanUseIPVSProxier and have it handle this case?